src/components/ProcessCreate/Steps/Confirm.tsx

```diff
@@ -32,6 +31,7 @@ import {
   UnpublishedElection,
   VocdoniCensus3Client,
   WeightedCensus,
+  ensure0x,
```

Comment:
@gerouvi please... do something with your import sorting. I think it's related to the fact you're using Typescript 5.X in vscode, instead of the one we're using in the project (4.9.5)